resources/migrations/59.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE gym
+ADD COLUMN `raid_pokemon_costume` smallint(4) unsigned DEFAULT 0;
```

Comment:
DEFAULT NULL

Reply:
ooops cant we just edit it right on the site? or just correct it on merge ~

Reply:
~ figured it out should be g2g